### DISCUSSION TITLE
8/7 안건

### DISCUSSION THREAD

**Discussion by gitul0515** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#119] [01.Agenda]
* 유저페이지 무한스크롤
* 동료의 작업물을 수정하고 싶을 때
* ExhibitionCard 이슈

**Comment by Hyevvy** [comment]
### 동료의 작업물을 수정하고 싶을 때 

- 담당자에게 건의한다 
- 해당 담당자가 동의하면 수정을 부탁 / 직접 작업해서 pr 올리기
